README.md

```diff
@@ -15,30 +15,27 @@ go get -u github.com/askmediagroup/dnsbench/cmd/dnsbench
 ## Usage and examples
 
 ```bash
-Execute a latency test.
+Execute a DNS benchmark test.
 
 Usage:
-  dnsbench run [command]
-
-Available Commands:
-  local       Benchmark the local system resolver configuration.
-  remote      Benchmark a remote nameserver.
+  dnsbench run [command] [flags]
 
 Flags:
-  -c, --concurrency int     Number of concurrent requests. (default 10)
-  -n, --count int           Total number requests. (default 1000)
+  -c, --count int           Number of queries to attempt. [0 = run until interrupted] (default 100)
   -h, --help                help for run
-  -i, --interval duration   Number of concurrent requests. (default 5s)
```

Comment:
ooh! was this a previous typo/mistake? @richardmarshall

Reply:
Yep, looks like an old copy&paste error.